src/modules/Shows.js

```diff
@@ -1,91 +1,115 @@
 class Shows {
-  static apiEndPoint = 'https://api.tvmaze.com/shows/1/episodes';
-
-  static getShows = async () => {
-    const response = await fetch(this.apiEndPoint).then((response) => response.json());
-    return response;
-  };
-
-  static updateUI = async () => {
-    const shows = await Shows.getShows();
-    const card = document.getElementById('card');
-    card.classList.add('wrap', 'flex');
-    const commentBody = document.getElementById('comment-section');
-
-    const tvShows = document.querySelector('#tv-shows p');
-    tvShows.innerHTML = `${'TV Shows ('}${shows.length})`;
-    shows.forEach((show, index) => {
-      card.innerHTML += `
-                  <div class="container flex-column">
-                      <div class="card-image"><img src="${show.image.medium}"></div>
-                      <div class="title-like">
-                          <div class="show-title"><p>${show.name}</p></div>
-                          <div class="likes">
-                              <div id='heart${show.id}' class="icon ptr"></div>
-                              <div class="likes-count"><p>5 likes</p></div>
+    static apiEndPoint = "https://api.tvmaze.com/shows/1/episodes";
+  
+    static getShows = async () => {
+      const response = await fetch(this.apiEndPoint).then((response) =>
+        response.json()
+      );
+      return response;
+    };
+  
+    static getLikes = async () => {
+      const action = {
+        method: "GET",
+        headers: {
+          "Content-type": "application/json; charset=UTF-8",
+        },
+      };
+      const response = await fetch(
+        "https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/tV364kOhzeIf5RoUn6sV/likes",
```

Comment:
While your implementation of this getLikes function is awesome, it would be even better if you separately define the string URL you used at the fetch method so that you will fill DRY principle and you will find it easy to use it next time for other functions.